lib/dpl/provider/lambda.rb

```diff
@@ -197,7 +228,19 @@ def default_kms_key_arn
       def function_tags
         options[:function_tags] ? split_string_array_to_hash(options[:function_tags]) : nil
       end
-
+      
+      def batch_size
+        options[:batch_size] ? options[:batch_size] : 1
```

Comment:
https://docs.aws.amazon.com/cli/latest/reference/lambda/create-event-source-mapping.html lists various values for the default value, depending on the back end. 1 seems a poor choice.